tests/unit/h/views/activity_test.py

```diff
@@ -1189,7 +1189,7 @@ def user_search_controller(self, user, pyramid_request):
 @pytest.fixture
 def group(factories):
     group = factories.Group()
-    group.members.extend([factories.User(), factories.User()])
+    group.members.extend([group.creator, factories.User(), factories.User()])
```

Comment:
This is a huge test file and (without spending a lot of time on it) it's hard to know which tests that use these shared fixtures actually depend on the creators being added to the groups as members. If they _aren't_ added then you get coverage failures where some lines of the test code aren't covered. I thought it best to just maintain the existing behaviors of these fixtures and add the creators as members.